opae.spec.in

```diff
@@ -104,6 +104,8 @@ ldconfig
 %files libs
 %defattr(-,root,root,-)
 @CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/libopae-c.so*
+@CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/libopae-c++.so*
```

Comment:
Don't include this version of C++ libraries. This is the older unofficial C++ library.

Reply:
okay..